static/templates/stream_settings/stream_subscription_request_result.hbs

```diff
@@ -3,10 +3,10 @@
 <br />
 {{/if}}
 {{#if subscribed_users}}
+    {{#if subscribed_users.[1]}} {{t "Successfully subscribed users:" }}
```

Comment:
How does this `.[1]` work? I'm not familiar with that syntax in Handlebars.

Reply:
> How does this `.[1]` work? I'm not familiar with that syntax in Handlebars.

I found this solution [here](https://stackoverflow.com/questions/28978759/length-check-in-a-handlebars-js-if-conditional). `.[1]` accesses the array value at the index 1.

> If it's an array, you don't need a helper! You can check if the item at length exists:
{{#if items.[20]}}